packages/libproj/meta.yaml

```
@@ -0,0 +1,44 @@
+package:
+  name: libproj
+  version: 8.2.1
+
+source:
+  sha256: 76ed3d0c3a348a6693dfae535e5658bbfd47f71cb7ff7eb96d9f12f7e068b1cf
+  url: https://download.osgeo.org/proj/proj-8.2.1.tar.gz
+
+requirements:
+  run:
+    - sqlite3
+
+build:
+  library: true
+  script: |
+    # Note: We need protoc binary during the build
+    case "$(uname -s)" in
+        Linux*) export MACHINE=linux;;
+        Darwin*) export MACHINE=osx;;
+        *) export MACHINE="UNKNOWN"
+    esac
+
+    export SQLITE_VERSION="3380500"
+    export SQLITE_ZIP="sqlite-tools-${MACHINE}-x86-${SQLITE_VERSION}.zip"
+    export SQLITE_URL="https://www.sqlite.org/2022/${SQLITE_ZIP}"
+    wget ${SQLITE_URL}
+    unzip ${SQLITE_ZIP} -d sqlite
+    export PATH=${PATH}:$(pwd)/sqlite/bin/
```

It looks like there is a subdirectory inside a sqlite zip file (maybe this is wrong, please check by yourself):

```suggestion
    export PATH=${PATH}:$(pwd)/sqlite/sqlite-tools-${MACHINE}-x86-${SQLITE_VERSION}
```